.all-contributorsrc

```diff
@@ -0,0 +1,24 @@
+{
+  "projectName": "gnuradio4",
+  "projectOwner": "fair-acc",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": ["README.md"],
+  "imageSize": 100,
+  "commit": false,
+  "contributorsPerLine": 7,
+  "contributorsSortAlphabetically": false,
```

Comment:
Just commenting since we were going with alphabetically for the list until now. I assume the non-alphabetical means sorted by time of being added to the contributions list? I'm okay with both ways.

Reply:
done